apps/render_cf.py

```diff
@@ -33,12 +37,11 @@ def get_env():
 
 
 def main():
-    parser = ArgumentParser()
-    parser.add_argument('--job-types-file', required=True)
-    job_types_file = parser.parse_args().job_types_file
-
-    with open(job_types_file) as f:
-        job_types = yaml.safe_load(f)
+    job_types = {}
+    job_spec_path = Path('./job_spec')
```

Comment:
Could reference this from `__file__` instead of `cwd` ... might make it so you could render from anywhere

Reply:
```suggestion
    job_spec_path = Path(__file__) / 'job_spec'
```

Reply:
I think you'll need `Path(__file__).parent / 'job_spec'`